packages/playground/src/dashboard/transfer_view.vue

```diff
@@ -21,6 +21,7 @@
                   validators.isNotEmpty('Invalid Twin ID'),
                   validators.isNumeric('Twin ID should be a number'),
```

Comment:
do we need isNumeric after having isInt?

Reply:
good catch :) validator removed.